omnibus/files/private-chef-cookbooks/private-chef/templates/default/rabbitmq.config.erb

```diff
@@ -0,0 +1,13 @@
+[
+ {ssl, [{versions, ['tlsv1.2', 'tlsv1.1']}]},
+ {rabbitmq_management,
+  [{listener, [{port,     <%= node['private_chef']['rabbitmq']['management_port'] %> },
```

Comment:
Will this by default bind to the same address as specified in ['rabbitmq']['vip']?